packages/ui-icons/source/index.js

```diff
@@ -0,0 +1,71 @@
+import React, {PropTypes} from 'react'
+import InlineSVG from 'react-svg-inline'
+import reduce from 'lodash/reduce'
+import find from 'lodash/find'
+import startCase from 'lodash/startCase'
+import {comp} from '@everydayhero/stranger'
+import ICONS from './icons'
+
+const SCALE_FACTOR = 1.25
+const kinds = ['flat', 'mono', 'emoji', 'sticker']
```

Comment:
Otherwise looks fine. Have you tested the library output? E.g. imported it locally into the Experience docs and rendered some icons?